src/payment/checkout/Checkout.jsx

```diff
@@ -130,10 +133,11 @@ class Checkout extends React.Component {
         <PaymentForm
           onSubmitPayment={this.handleSubmitCybersource}
           onSubmitButtonClick={this.handleSubmitCybersourceButtonClick}
-          disabled={submissionDisabled}
+          disabled={editable}
           loading={loading}
           isProcessing={cybersourceIsSubmitting}
           isBulkOrder={isBulkOrder}
+          isQuantityUpdating={bulkQuantityIsUpdating}
```

Comment:
Nit: Personally, if we choose to use this prop, I would name it the same thing in both `PaymentForm` and `Checkout`

Reply:
Noted, will do. I followed the `cybersourceIsSubmitting` but also prefer naming the same.